doc/submitter_guide.md

```diff
@@ -60,19 +60,18 @@ dashboard](https://kcidb.kernelci.org/).
 2\. Install KCIDB
 -----------------
 
-Pick a KCIDB [release][releases]. Latest would normally be best.
 
-Let's say you picked v9. Run this:
+This will install the latest version of the package from the main branch of the kcidb repository.
 
 ```bash
-pip3 install --user 'git+https://git@github.com/kernelci/kcidb.git@v9'
+pip3 install --user 'git+https://git@github.com/kernelci/kcidb.git@main'
 ```
 
 Then make sure your PATH includes the `~/.local/bin` directory, e.g. with:
 
     export PATH="$PATH":~/.local/bin
 
-See [Installation](../installation) for alternatives, and if you know your
+See [Installation](installation.md) for alternatives, and if you know your
```

Comment:
Unfortunately, this won't work when deployed to https://kernelci.org/docs/kcidb/submitter_guide/ Please revert.

Reply:
Okay done